dayrepo/snippets/templates/car_add.html

```diff
@@ -0,0 +1,22 @@
+{% load static %}
+<link rel="stylesheet" href='{% static "css/style.css" %}'>
+<h2>車両情報設定</h2>
+<form method="post">
+    <p>
+        <label for="id_vehicle_number">車両番号:</label>
+        <input type="text" name="vehicle_number_place" class="vehicle_number_place" maxlength="10" required="" id="id_vehicle_number" placeholder=札幌>
+        <input type="number" name="vehicle_number_type" class="vehicle_number_type" maxlength="3" required="" id="id_vehicle_number" placeholder=100>
+        <input type="text" name="vehicle_number_use" class="vehicle_number_use" maxlength="1" required="" id="id_vehicle_number" placeholder=あ>
```

Comment:
required 属性は、基本的に何かを代入するような属性じゃないのと、こう書いてしまうと必須項目になっていないのではないでしょうか？動作確認はしてみましたか？（単純にやったことないので気になる
https://developer.mozilla.org/ja/docs/Web/HTML/Attributes/required
```suggestion
        <input type="text" name="vehicle_number_use" class="vehicle_number_use" maxlength="1" required　id="id_vehicle_number" placeholder=あ>
```
※他のタグも同様です

Reply:
必須項目として機能しています。
![image](https://github.com/bob-g12/dayrepo-for-transportation/assets/126374166/1fd9d54e-8edd-4776-a79d-3bae522a10a9)

Reply:
=""があっても機能しましたが、削除しておきました。
https://github.com/bob-g12/dayrepo-for-transportation/pull/70/commits/663573ff0ba6b20e5e552f14e5e66beb55b76bd6